src/commands/deploy.js

```diff
@@ -4,7 +4,8 @@ const fs = require('fs');
 const getAppInfo = require('../prompts/getAppInfo');
 const Conf  = require('conf');
 const config = new Conf();
-const template = fs.readFileSync('./src/utils/bastion.yaml', 'utf8');
+const path = require("path");
+const template = fs.readFileSync(path.resolve(__dirname, '../utils/bastion.yaml'), 'utf8');
```

Comment:
This was causing cli app to crash when I try to run it from another folder in the terminal